ts/features/pn/components/MessageDetails.tsx

```diff
@@ -60,6 +60,7 @@ export const MessageDetails = ({
   service,
   payments
 }: Props) => {
+  // console.log(`=== MessageDetails: rendering`);
```

Comment:
Why we need this?
```suggestion
```

Reply:
We do not. They are there to facilitate the final testing, when we will have everything in place and need to check the entire implementation (we are still 3-4 PRs behind) for re-renderings